projects/portal/src/app/pages/mis-turnos/mis-turnos.component.ts

```diff
@@ -23,7 +22,7 @@ export class PDPMisTurnosComponent implements OnInit {
         private el: ElementRef) { }
 
     ngOnInit(): void {
-        this.turnos$ = this.turnoService.getTurnos().pipe(
+        this.turnos$ = this.turnoService.turnosFiltrados$.pipe(
             map(turnos => this.sortTurnos(turnos))
         );
```

Comment:
Cuando hay mas de una subscripcion, conviene ponerle cache al observable
```suggestion
        this.turnos$ = this.turnoService.turnosFiltrados$.pipe(
            map(turnos => this.sortTurnos(turnos)),
            cache()
        );
```

Reply:
Bien ahi, se me pasó.